### PR TITLE
Feature/unit test

### PR DESCRIPTION
**Pull request by jesuspg**
@pratid

### CODE REVIEW

**Comment by jesuspg** [comment]
https://coveralls.io/builds/2794223

File: dashboard/lib/app.js [hunk 1]
```diff
@@ -40,23 +40,38 @@ var express = require('express'),
 var app = express();
 
 
-logger.info('Running app in web context: '+config.web_context);
-app.base=config.web_context;
+logger.info('Running app in web context: ' + config.web_context);
```
Comment (pratid) [inline dashboard/lib/app.js]: Use string interpolation instead of concatenation:
`logger.info('Running app in web context: %s', config.web_context)`

Comment (jesuspg) [inline dashboard/lib/app.js]: :+1:

File: dashboard/package.json [hunk 2]
```diff
@@ -39,7 +39,8 @@
     "stylus": "^0.51.1",
     "underscore": "^1.8.3",
     "util": "^0.10.3",
-    "js-yaml": "^2.0.5"
+    "js-yaml": "^2.0.5",
+    "sinon": "^1.15.3"
```
Comment (pratid) [inline dashboard/package.json]: sinon should be added as "devDependency" (next time run `npm install xxxx --save-dev`)

**Comment by pratid** [comment]
LGTM

**Pull request merged**